src/controllers/productController.ts

```diff
@@ -7,6 +7,7 @@ import {
 } from '../config/index'
 import Product from '../database/models/productModel'
 import Collection from '../database/models/collectionModel'
+import { deleteProductById, getProductById } from '../services/productServices'
```

Comment:
```suggestion
import { deleteProductById, getProductById } from '../services/productServices'
```
is productServices or productService